test/e2e/specs/editor/blocks/links.spec.js

```diff
@@ -58,7 +58,72 @@ test.describe( 'Links', () => {
 		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
 	} );
 
-	test.describe( 'should contain a label when it should open in a new tab', () => {} );
+	test( 'should contain a label when it should open in a new tab', async ( {
```

Comment:
I know you copy/pasted this but this test name is not very descriptive. I cannot really determine what this is testing from the test name.

As far as I can see we're not even asserting on things to do with label?